appendix.md

```diff
@@ -0,0 +1,34 @@
+# Appendix
+
+## ZMQ
+
+Some useful hints regarding the working of the zmq package.
+
+(router-sockets)=
+### Particularities of ROUTER sockets
+
+A small introduction to ROUTER sockets, for more details see [zmq guide chapter 3](https://zguide.zeromq.org/docs/chapter3/#Exploring-ROUTER-Sockets).
+
+A router socket assigns a random _identity_ to each connecting peer.
```

Comment:
I disagree. There are several more socket types.

> Then, it  assigns a random _identity_ to each connecting peer.

Even if a ROUTER socket connects to some other socket, it assigns an _identity_ to that connection.

Reply:
yes, of course, there are many more socket types, which typically serve special purposes. The ROUTER socket can be connected with most (if not all, not sure about PUB-SUB) socket types, however that does not mean that the other socket types are useless. IIRC, ROUTERs cannot initiate communication as long as they do not know an _identity_ to which they could send messages. If the application code knows the _identity_ from some other source, the ROUTER can start, but otherwise it cannot. 
This is all I wanted to put in there, to give a bit more context.

Reply:
ROUTER cannot connect to PUB-SUB, nor PAIR, nor RADIO, nor DISH...
Only to ROUTER, REP, REQ, DEALER.

Anyways, I tried to make it more clear in my change (coming soon).

Reply:
Okay, I have to admit, RADIO-DISH and SERVER-CLIENT socket connections were now new to me, and PAIR was somehow always kind of out of my scope. I like your change!